bundles/org.openhab.binding.upnpcontrol/src/main/java/org/openhab/binding/upnpcontrol/internal/handler/UpnpServerHandler.java

```diff
@@ -207,7 +207,12 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                                 // No parent found, so make it the root directory
                                 browseTarget = DIRECTORY_ROOT;
                             }
-                            currentEntry = parentMap.get(browseTarget);
+                            UpnpEntry entry = parentMap.get(browseTarget);
+                            if (entry == null) {
+                                return;
```

Comment:
Should probably log something here, looks like there is an expectation for the `browseTarget` to be logged for debugging. I'd suggest logging a message here regarding the lack of an entry for that browseTarget.